src/apply.ml

```diff
@@ -0,0 +1,31 @@
+open Basics
```

Comment:
I'm not too happy with moving stuff to new files in this way. First, it does not really make sense to have 30 lines files, and second, this introduces code duplication. The same changes would only introduce three or four lines in the current `src/handle.ml`.

Note that `src/handle.ml` will soon be split in two, but I'd rather do these changes myself since I have a general plan in mind. But at least, we should discuss this before working in two different directions.